lib/phantomjs/platform.rb

```diff
@@ -9,10 +9,6 @@ def architecture
         RbConfig::CONFIG['host_cpu']
       end
 
-      def temp_path
-        ENV['TMPDIR'] || ENV['TEMP'] || '/tmp'
```

Comment:
Anyway I wonder why this custom dir lookup is used instead of [Dir#tmpdir lookup](https://github.com/ruby/ruby/blob/1b34bf1943b18e9baf48902c81151253bfc55280/lib/tmpdir.rb#L26).